### PR TITLE
Event Detail Page

### PR DESCRIPTION
**Pull request by radscheit**

### CODE REVIEW

File: app/assets/stylesheets/custom_bootstrap/custom_bootstrap.less [hunk 1]
```diff
@@ -163,6 +163,18 @@ a:hover h1, a:hover h2, a:hover h3, a:hover h4 {
 	height: 20px;
 	min-height: 20px; 
 }
+
+.spacer-30	{
```
Comment (cmfcmf) [inline app/assets/stylesheets/custom_bootstrap/custom_bootstrap.less]: nit: remove tab

File: app/assets/stylesheets/custom_bootstrap/custom_bootstrap.less [hunk 2]
```diff
@@ -163,6 +163,18 @@ a:hover h1, a:hover h2, a:hover h3, a:hover h4 {
...
+	width: 100%;
+	height: 30px;
+	min-height: 30px; 
+}
+
+.spacer-60	{
```
Comment (cmfcmf) [inline app/assets/stylesheets/custom_bootstrap/custom_bootstrap.less]: nit: remove tab

File: app/assets/stylesheets/custom_bootstrap/custom_bootstrap.less [hunk 3]
```diff
@@ -253,3 +263,30 @@ strong, b {
...
+}
+
+.events-facts {
+	ul {
+		padding-left: 25px;
+	}
+}
+
+.bottom-align-text {
+ position: absolute;
```
Comment (cmfcmf) [inline app/assets/stylesheets/custom_bootstrap/custom_bootstrap.less]: nit: missing indent

File: app/views/events/_material_area.html.erb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <% if can? :view_material, Event %>
-  <div class="col-xs-12 col-sm-12 col-md-10 col-lg-10 col-xs-offset-0 col-sm-offset-0 col-md-offset-1 col-lg-offset-1" id="event_material_area">
+  <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12" id="event_material_area">
```
Comment (cmfcmf) [inline app/views/events/_material_area.html.erb]: You can remove everything except for `col-xs-12`.

File: app/views/events/_material_area.html.erb [hunk 5]
```diff
@@ -27,7 +27,9 @@
       <% end %>
       </tbody>
     </table>
-
+  </div>
+  <div class="spacer-60"></div>
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6 col-xs-offset-0 col-sm-offset-3 col-md-offset-3 col-lg-offset-3" id="event_material_area">
```
Comment (cmfcmf) [inline app/views/events/_material_area.html.erb]: Remove everything except for `col-xs-12 col-sm-6 col-xs-offset-0 col-sm-offset-3`

File: app/views/events/_show_event_large.html.erb [hunk 6]
```diff
@@ -1,46 +1,62 @@
...
     <%= link_to t('.edit', :default => t('helpers.links.edit')), edit_event_path(@event),
                 :class => 'btn btn-default btn-sm' if can? :edit, @event %>
   </div>
-  <h1><%= @event.name %></h1>
 </div>
 <div class="row">
-  <div class="col-md-1 col-md-offset-1 text-center col-sm-2">
+  <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 line-spacer">
+  </div>
+</div>
```
Comment (cmfcmf) [inline app/views/events/_show_event_large.html.erb]: What is this whole row? Just for spacing? Use a single `<div>` without class instead.

**Review by cmfcmf — CHANGES_REQUESTED** [review]
You seem to always specify all breakpoints for all viewports. I personally think this is unneeded and using `col-xs-12` is cleaner than using `col-xs-12 col-sm-12 col-md-12 col-lg-12`.

File: app/views/events/_material_area.html.erb (hunk 4 above)
Comment (radscheit) [reply app/views/events/_material_area.html.erb]: Yip I use Zen-Coding, so the classes were added automatically, but I see your point

File: app/views/events/_show_event_large.html.erb [hunk 7]
```diff
@@ -1,46 +1,62 @@
...
+  <div class="col-xs-12 col-sm-3 text-right bottom-align-text">
     <%= render :partial => 'apply_button', :locals => {event: @event} %>
     <%= link_to t('.edit', :default => t('helpers.links.edit')), edit_event_path(@event),
                 :class => 'btn btn-default btn-sm' if can? :edit, @event %>
   </div>
-  <h1><%= @event.name %></h1>
 </div>
 <div class="row">
-  <div class="col-md-1 col-md-offset-1 text-center col-sm-2">
+  <div class="line-spacer">
```
Comment (cmfcmf) [inline app/views/events/_show_event_large.html.erb]: The enclosing row can also be removed

File: app/views/events/_show_event_large.html.erb [hunk 8]
```diff
@@ -1,46 +1,62 @@
...
-      <% end %>
-    </dl>
+  <div class="col-md-3">
+    <div class="panel panel-primary events-facts">
+      <div class="panel-heading">
+        <%= t('events.facts') %>
+      </div>
+      <div class="panel-body">
+        <% if not @event.knowledge_level.blank? or can? :manage, Event %>
+          <p><h5><%= model_class.human_attribute_name(:knowledge_level) %></h5></p>
```
Comment (cmfcmf) [inline app/views/events/_show_event_large.html.erb]: `<h5>` inside `<p>` is wrong.

**Review by cmfcmf — CHANGES_REQUESTED** [review]
Found a few more minor markup problems.

**Review by cmfcmf — APPROVED** [review]
LGTM now 👍

**Pull request merged**